GOES/Scripts/plot_goes_data.py

```diff
@@ -100,82 +70,48 @@ def plot_goes_data():
     input:  none but read from: 
         https://services.swpc.noaa.gov/json/goes/primary/differential-protons-3-day.json
         https://services.swpc.noaa.gov/json/goes/primary/integral-protons-3-day.json
-    output: <plot_dir>/goes_perticls.png
+    output: <plot_dir>/goes_particles.png
             <plot_dir>/goes_proton.png
     """
-    ddata = extract_goes_data(dlink, diff_list, 1e3)
+    ddata = extract_goes_data(DLINK, DIFF_LIST, 1e3)
```

Comment:
Note that this factor is intended to multiply the flux value to convert units from KeV to MeV. However as a multiplicative factor, it's actually converting KeV to eV and should instead be 1e-3. I have concerns later on algorithmically about whether this factor is needed here but if it turns out it's till needed, I'll correct it to 1e-e3